// function fun() {
//   return new Promise((resolve, reject) => {
//     const error = false;
//     setTimeout(() => {
//       if (error) {
//         reject("Promise Rejected");
//         return;
//       }
//       resolve("Promise Resolved");
//     }, 2000);
//   });
// }

// fun()
//   .then((res) => console.log(res))
//   .catch((err) => console.log(err));
